### PR TITLE
fix: add label for point event detect

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* Add Area chart point type event detection
* Add label for point event detect

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다!

File: apps/chart/src/component/areaSeries.ts [hunk 1]
```diff
@@ -213,21 +214,45 @@ export default class AreaSeries extends Component {
...
+    categories: string[],
+    rawCategories: string[],
+    axisData: LabelAxisData
+  ) {
+    if (this.eventDetectType === 'near') {
+      return this.makeNearTypeResponderModel(responderModel, tooltipDataArr, rawCategories);
+    }
+    if (this.eventDetectType === 'point') {
+      return this.makeNearTypeResponderModel(responderModel, tooltipDataArr, rawCategories, 0);
+    }
```
Comment (js87zz) [inline apps/chart/src/component/areaSeries.ts:238]: 이런 경우는 인자를 객체로 받는게 더 명확하지 않을까요?

File: apps/chart/src/component/areaSeries.ts [hunk 2]
```diff
@@ -158,6 +158,7 @@ export default class AreaSeries extends Component {
     let areaStackSeries;
     const options = this.getAreaOptions(chartState.options);
     const categories = chartState.categories as string[];
+    const rawCategories = (chartState.rawCategories as string[]) ?? [];
```
Comment (js87zz) [inline apps/chart/src/component/areaSeries.ts:161]: 어썰션으로 강제 변환해줄 수 밖에 없는건가요~?

**Review by js87zz — APPROVED** [review]
리뷰 완료입니다!

**Review by jungeun-cho — APPROVED** [review]
`eventDetectType: 'point',  zoomable: true, selectable: true`했을 때 selectable이 정상적으로 되지 않는 현상이 있었는데 정상동작 확인하였습니다.
수고하셨어요~

File: apps/chart/src/component/areaSeries.ts (hunk 2 above)
Comment (jung-han) [reply apps/chart/src/component/areaSeries.ts:161]: 타입이 x, y카테고리가 들어가는 경우와 있어서 타입을 분리하지 않는한 타입을 변환해주는게 필요합니다ㅠ

**Pull request merged**